filter-geoip/geoip.go

```
@@ -93,13 +95,16 @@ func (p *processor) Configure(ctx veino.ProcessorContext, conf map[string]interf
 
 	p.cache = lrucache.New(p.opt.CacheSize)
 	p.databases = map[string]*geoip2.Reader{}
-	p.load()
-	return p.ConfigureAndValidate(ctx, conf, p.opt)
 
+	if err := p.load(conf["databases"]); err != nil {
+		log.Fatalln(err)
```

I'm not really happy with using default log... I tried to use p.Logger.Fatalln() but then it is silently ignored :(